src/test/java/org/opensearch/security/InitializationIntegrationTests.java

```diff
@@ -120,14 +122,39 @@ public void testWhoAmI() throws Exception {
         try (RestHighLevelClient restHighLevelClient = getRestClient(clusterInfo, "spock-keystore.jks", "truststore.jks")) {
             Response whoAmIRes = restHighLevelClient.getLowLevelClient().performRequest(new Request("GET", "/_plugins/_security/whoami"));
             Assert.assertEquals(whoAmIRes.getStatusLine().getStatusCode(), 200);
+            // Should be using HTTP/2 by default
+            Assert.assertEquals(whoAmIRes.getStatusLine().getProtocolVersion(), HttpVersion.HTTP_2);
             JsonNode whoAmIResNode = DefaultObjectMapper.objectMapper.readTree(whoAmIRes.getEntity().getContent());
             String whoAmIResponsePayload = whoAmIResNode.toPrettyString();
             Assert.assertEquals(whoAmIResponsePayload, "CN=spock,OU=client,O=client,L=Test,C=DE", whoAmIResNode.get("dn").asText());
             Assert.assertFalse(whoAmIResponsePayload, whoAmIResNode.get("is_admin").asBoolean());
             Assert.assertFalse(whoAmIResponsePayload, whoAmIResNode.get("is_node_certificate_request").asBoolean());
         }
     }
-    
+
+    @Test
```

Comment:
Curious, is there a way for a cluster admin to force HTTP/1? Would that ever be valid or desired as a config setting?

Reply:
Hm ... this is a good question, I don't see why one would do that intentionally, but the `RestClientBuilder` could force HTTP/1.1, see please [1] fe

[1] https://github.com/opensearch-project/security/pull/2367/files#diff-25d921c48bb0ef8fb56806d0d9cbed23e72f8e9b275a9ec606bc6be78cecb37eR192